From ef0b5a2866d899a80078c3d35c760fbe1369f1b5 Mon Sep 17 00:00:00 2001
From: Sagar Bussa <sagar.bussa@morsemicro.com>
Date: Mon, 10 Mar 2025 19:50:55 +1100
Subject: [PATCH] kernel: enable dev coredump by default

A Morse Micro driver compiled out of tree has no way to signal to the
kernel config to enable WANT_DEV_COREDUMP. Enable it by default.
---
 drivers/base/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/base/Kconfig b/drivers/base/Kconfig
index 064eb52ff7e2d..5a89788472ea1 100644
--- a/drivers/base/Kconfig
+++ b/drivers/base/Kconfig
@@ -105,6 +105,7 @@ source "drivers/base/firmware_loader/Kconfig"
 
 config WANT_DEV_COREDUMP
 	bool
+	default y
 	help
 	  Drivers should "select" this option if they desire to use the
 	  device coredump mechanism.
